verification/curator-service/ui/src/components/CaseForm.tsx

```diff
@@ -405,7 +409,11 @@ class CaseForm extends React.Component<Props, CaseFormState> {
                 }),
             symptoms: {
                 status: unknownToUndefined(values.symptomsStatus),
-                values: values.symptoms,
+                values:
+                    values.symptomsStatus === 'Symptomatic' ||
```

Comment:
Am i understanding correctly that you could select Symptomatic, then select symptoms, then remove the Symptomatic selection but symptoms would still be sent? If this wasn't happening I'm not sure what we're getting by doing this.

Reply:
Yes that was happening. A big bug!